libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -1354,4 +1393,9 @@ class NavEKF3_core : public NavEKF_core_common
     */
     void updateMovementCheck(void);
 
+    // The following declarations are used to control when the main navigation filter resets it's yaw to the estimate provided by the GSF
+	uint64_t EKFGSF_yaw_reset_ms;	        // timestamp of last emergency yaw reset (uSec)
```

Comment:
these are being initialised from imuSampleTime_ms which are uint32_t, so should also be uint32_t

Reply:
will fix